test/test-utils.js

```diff
@@ -331,3 +337,123 @@ export const adminCreateCategory =
     }
     return res;
   };
+export const createVoucherType = (chain) => async (description, duration) => {
+  const VOUCHER_HUB_ABI = {
+    inputs: [
+      {
+        internalType: 'string',
+        name: 'description',
+        type: 'string',
+      },
+      {
+        internalType: 'uint256',
+        name: 'duration',
+        type: 'uint256',
+      },
+    ],
+    name: 'createVoucherType',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  };
+  const voucherHubContract = new Contract(
+    chain.voucherHubAddress,
+    VOUCHER_HUB_ABI,
+    chain.voucherManagerWallet,
+  );
+  return voucherHubContract.createVoucherType(description, duration);
+};
+
+// TODO: update createWorkerpoolorder() parameters when it is specified
+const createAndPublishWorkerpoolOrder = async (chain, workerpool, user) => {
+  const iexec = new IExec(
+    {
+      ethProvider: getSignerFromPrivateKey(
+        chain.rpcURL,
+        chain.prodWorkerpoolOwnerWallet.privateKey,
+      ),
+    },
+    { hubAddress: chain.hubAddress },
+  );
+  const prodWorkerpoolOrder = await iexec.createWorkerpoolorder({
+    workerpool,
+    category: 0,
+    requesterrestrict: user,
+    volume: 1000,
+    workerpoolprice: 10000000000000000,
+  });
+  await iexec.publishWorkerpoolorder(prodWorkerpoolOrder);
+};
+
+export const createVoucher = async (chain, { owner, voucherType, value }) => {
+  const VOUCHER_HUB_ABI = {
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'owner',
+        type: 'address',
+      },
+      {
+        internalType: 'uint256',
+        name: 'voucherType',
+        type: 'uint256',
+      },
+      {
+        internalType: 'uint256',
+        name: 'value',
+        type: 'uint256',
+      },
+    ],
+    name: 'createVoucher',
+    outputs: [
+      {
+        internalType: 'address',
+        name: 'voucherAddress',
+        type: 'address',
+      },
+    ],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  };
```

Comment:
abi must be an array of abi fragments

```suggestion
  const VOUCHER_HUB_ABI = [{
    inputs: [
      {
        internalType: 'address',
        name: 'owner',
        type: 'address',
      },
      {
        internalType: 'uint256',
        name: 'voucherType',
        type: 'uint256',
      },
      {
        internalType: 'uint256',
        name: 'value',
        type: 'uint256',
      },
    ],
    name: 'createVoucher',
    outputs: [
      {
        internalType: 'address',
        name: 'voucherAddress',
        type: 'address',
      },
    ],
    stateMutability: 'nonpayable',
    type: 'function',
  }];
```